script/setup/install-critest

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# Builds and installs cni plugins to /opt/cni/bin,
+# and create basic cni config in /etc/cni/net.d.
+# The commit defined in vendor.conf
+#
+set -eu -o pipefail
+
+CRITEST_COMMIT=v1.0.0-alpha.0
```

Comment:
is there a better place to get the version tag from? maybe a todo or issue

Reply:
Seems not. :P I think it's fine to hard code it for now.